lib/assets/stylesheets/modules/_technical-documentation.scss

```diff
@@ -226,16 +200,20 @@
     p+p, p+ul, p+ol,
     ul+p, ul+ol,
     ol+p {
-      margin-top: $gutter-one-third;
+      margin-top: govuk-spacing(2);
     }
   }
 
   hr {
     height: 1px;
     border: none;
-    margin-top: $gutter - 1px;
-    margin-bottom: $gutter;
-    background-color: $border-colour;
-    color: $border-colour;
+    margin-top: govuk-spacing(6) - 1px;
+    margin-bottom: govuk-spacing(6);
+    background-color: $govuk-border-colour;
+    color: $govuk-border-colour;
+  }
+
+  a {
```

Comment:
I think this is duplicating what's on line 66:

https://github.com/alphagov/tech-docs-gem/pull/97/files#diff-d7f12a4697fd2195a8c62339caee5c77R66-R68